Formula/ffmpeg.rb

```diff
@@ -3,9 +3,9 @@ class Ffmpeg < Formula
   homepage "https://ffmpeg.org/"
   url "https://ffmpeg.org/releases/ffmpeg-4.3.1.tar.xz"
   sha256 "ad009240d46e307b4e03a213a0f49c11b650e445b1f8be0dda2a9212b34d2ffb"
-  # None of these parts are used by default, you have to explicitly pass `--enable-gpl`
-  # to configure to activate them. In this case, FFmpeg's license changes to GPL v2+.
-  license "GPL-2.0"
+  # FFmpeg's license changes to GPL v2+, as --enable-gpl is always passed.
```

Comment:
shall i also link to `https://ffmpeg.org/legal.html` in this comment?

Reply:
could even add something like `to build FFmpeg without this flag (resulting LGPLv2.1+ license), use homebrew-ffmpeg/ffmpeg/ffmpeg`

Reply:
Please just change the license identifier.

Reply:
@SMillerDev sure, reverted 🤙